luna/pretrained_models/googlenet.py

```diff
@@ -430,8 +431,17 @@ def create_googlenet():
                                              loss2_classifier_act,
                                              loss3_classifier_act])
 
-    googlenet.load_weights(os.path.join(
-        os.path.dirname(__file__), 'googlenet_weights.h5'))
+    weights_path = Path(str(Path.home()) + r"\.keras\datasets\googlenet_weights.h5")
```

Comment:
Same as above, thee assumptions could be problematic, so I would advocate for removing them and have the user specify weights path.

Reply:
Weights path is still an optional variable but if it is not given, it will be downloaded. It is directory independent so should not give problem in different OS.